src/main/webapp/join-room.test.js

```diff
@@ -0,0 +1,7 @@
+const script = require('./join-room');
+
+test('get roomId from given url', () => {
```

Comment:
This is a good start so far. In addition to this case, I think it would be good to test a few other cases, e.g. URLs that are missing roomId params, URLs that don't have a "?", etc.

Reply:
It's looking a lot better now, thanks for adding the different test cases. Like Aleksey said, each test case should have a description that describes what it's testing, rather than all of them having the same description